lib/src/chart/pie_chart/pie_chart_connector_lines.dart

```diff
@@ -0,0 +1,18 @@
+import 'package:flutter/material.dart';
+
+/// Type of connector line.
+enum FLConnectorType { line }
```

Comment:
I didn't understand the use case of this enum
We have FLConnectorLineSettings (which is a line itself as the name suggests), why we need a FLConnectorType (which is again a line)?

Reply:
I added this to keep an idea for those who wanted enhance this more, for example, might be another enum `.dotted`, `.dashed` and so on. I'll remove it in the next commit to clean it make it ready for merge. Thanks for highlighting.